terraform/environments/hmpps-domain-services/locals_test.tf

```diff
@@ -182,60 +181,268 @@ locals {
       }
     }
 
+    baseline_ec2_instances = {
+      test-rds-1-b = {
+        # ami has unwanted ephemeral device, don't copy all the ebs_volumess
+        config = merge(module.baseline_presets.ec2_instance.config.default, {
+          ami_name                      = "hmpps_windows_server_2022_release_2023-12-02T00-00-15.711Z"
```

Comment:
any reason why these amis references can't be wildcarded in case they get rebuilt?